torchvision/io/_video_opt.py

```diff
@@ -378,9 +378,15 @@ def _read_video_from_memory(
         the audio frames, where `L` is the number of points and
             `K` is the number of channels
     """
-
-    _validate_pts(video_pts_range)
-    _validate_pts(audio_pts_range)
+    if video_pts_range is None:
+        video_pts_range = [0, -1]
+    else:
+        _validate_pts(video_pts_range)
+
+    if audio_pts_range is None:
+        audio_pts_range = [0, -1]
+    else:
+        _validate_pts(audio_pts_range)
```

Comment:
Let's keep the old behavior more consistent by always calling `_validate_pts`
```suggestion
    if video_pts_range is None:
        video_pts_range = [0, -1]

    if audio_pts_range is None:
        audio_pts_range = [0, -1]

    _validate_pts(video_pts_range)
    _validate_pts(audio_pts_range)
```